{{cookiecutter.repo_name}}/README.rst

```diff
@@ -152,8 +152,32 @@ The email server listens on 127.0.0.1:1025
 It's time to write the code!!!
 
 
+Running end to end integration tests
+------------------------------------
+
+N.B. The integration tests will not run on Windows.
+
+To install the test runner::
+
+  $ sudo pip install hitch
```

Comment:
I suppose that sudo is not necessary, unless you really want to install hitch globally